client/src/components/CreateMeetup/CreateMeetup.js

```diff
@@ -210,11 +254,25 @@ function CreateMeetup() {
     );
   };
 
+  const onSubmit = () => {
+    setCreatingMeetup(true);
+    if (!validateForm()) {
+      setCreatingMeetup(false);
+      return;
+    }
+    createMeetup();
```

Comment:
```suggestion
    if (creatingMeetup || !validateForm()) {
      return;
    }
    setCreatingMeetup(true);
    createMeetup();
```

Also, feel free to just inline the createMeetup function here. No need for it to be a separate function.